themes/scss/vendor/leaflet.scss

```diff
@@ -278,12 +278,12 @@
 	border-radius: 5px;
 	}
 .leaflet-control-layers-toggle {
-	background-image: url(images/layers.png);
+	background-image: url(../../img/layers.png);
 	width: 36px;
 	height: 36px;
 	}
 .leaflet-retina .leaflet-control-layers-toggle {
-	background-image: url(images/layers-2x.png);
+	background-image: url(../../img/layers-2x.png);

```

Comment:
Line should be indented with spaces, not tabs<br>URLs should be enclosed in quotes